AGENTS.md

```
@@ -386,81 +426,74 @@ bun run dev
 
 ### Hono Adapter
 
-**createAgentApp(meta, options?)**
+**createAgentApp(runtimeOrBuilder)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)